### DISCUSSION TITLE
Logitech (Saitek) Multipanel not work with new AP (Experimental version)

### DISCUSSION THREAD

**Discussion by kolyadkl** [flybywiresim/aircraft#4212] [General]
Hello guys! First of all, sorry for my English. Can I ask - anybody use Multipanel (AP) with Experimental version. It worked perfect, but after update (with new AP, if i understand right) now it work with many bugs - speed not change, vertical speed not working at all and other. Is there some kind of solution?

**Comment by tracernz** [comment]
Same issue in https://github.com/flybywiresim/a32nx/issues/4180 I think.

**Reply by kolyadkl** [reply]
If I understand correctly - just wait now?

**Reply by tracernz** [reply]
Yes. The experimental branch is experimental and under development, so some things are expected to be broken until it’s ready for merge into master.
